### PR TITLE
Apply jobs page creation 

### PR DESCRIPTION
**Pull request by Mohit-Sony**

### CODE REVIEW

File: components/Careers/ApplyNewJob.vue [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+                    <li class="inline text-slate-400 ltr:mr-3 rtl:ml-3"><i class="uil uil-map-marker text-white h6 ltr:mr-1 rtl:ml-1"></i> London, UK - <span class="text-white">Full Time</span></li>
+                </ul>
+            </div>
+            <!--end grid-->
+        </div>
+        <!--end container-->
+    </section>
+    <!--end section-->
+    <div class="relative">
+        <div class="shape absolute right-0 sm:-bottom-px -bottom-[2px] left-0 overflow-hidden text-white dark:text-slate-900">
```
Comment (panthsolanki) [inline components/Careers/ApplyNewJob.vue]: Can you please check If theme can change from header then you can remove from here

File: components/Careers/ApplyNewJob.vue [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+                                <div class="lg:col-span-6">
+                                    <div class="ltr:text-left rtl:text-right">
+                                        <FormField label="Name" help="Required. Your name" :error="errorMsgs.name">
+                                            <FormControl
+                                              v-model="formData.name"
+                                              name="username"
+                                              required
+                                              autocomplete="username"
+                                            />
+                                          </FormField>                                    </div>
```
Comment (panthsolanki) [inline components/Careers/ApplyNewJob.vue]: Please do proper code formatting, can you use esLint extension

File: components/Careers/ApplyNewJob.vue [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+                                          </FormField>                                      </div>
+                                </div>
+
+                                <div class="lg:col-span-12">
+                                        <FormField label="Upload file:" help="Max 500kb">
+                                            <FormFilePicker label="Upload" />
+                                        </FormField>
+                                </div>
+                            </div>
+                            <button type="submit" @click.prevent="submitHandler" id="submit" name="send" class="btn bg-indigo-600 hover:bg-indigo-700 border-indigo-600 hover:border-indigo-700 text-white rounded-md mt-4">Send Message</button>
```
Comment (panthsolanki) [inline components/Careers/ApplyNewJob.vue]: do code formatting

File: components/Careers/ApplyNewJob.vue [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+  mdiAccount,
+  mdiMail,
+  mdiAsterisk,
+  mdiFormTextboxPassword,
+  mdiCreditCardOutline,
+} from "@mdi/js";
+import FormField from "@/components/Forms/FormField.vue";
+import FormControl from "@/components/Forms/FormControl.vue";
+import FormFilePicker from "@/components/Forms/FormFilePicker.vue";
+// import
```
Comment (panthsolanki) [inline components/Careers/ApplyNewJob.vue]: remove unwanted comment

File: components/Careers/ApplyNewJob.vue [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+                                              name="email"
+                                              required
+                                              autocomplete="email"
+                                            />
+                                          </FormField>                                      </div>
+                                </div>
+
+                                <div class="lg:col-span-12">
+                                        <FormField label="Upload file:" help="Max 500kb">
+                                            <FormFilePicker label="Upload" />
```
Comment (panthsolanki) [inline components/Careers/ApplyNewJob.vue]: add constraint  of only pdf file under 5MB should be selected

File: components/Careers/ApplyNewJob.vue [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+        if(!val ){
+            errorMsgs.value[field] = "Required field"
+        return false;
+      }
+      errorMsgs.value[field] = ""
+      return true;
+    };
+
+    const validateEmail = (email) => {        
+      const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
```
Comment (panthsolanki) [inline components/Careers/ApplyNewJob.vue]: CAN: update regex as this will take special characters

File: components/Careers/ApplyNewJob.vue (hunk 1 above)
Comment (Mohit-Sony) [reply components/Careers/ApplyNewJob.vue]: yes its working fine from header. i have removed it from here

File: components/Careers/ApplyNewJob.vue (hunk 2 above)
Comment (Mohit-Sony) [reply components/Careers/ApplyNewJob.vue]: have a look

File: components/Careers/ApplyNewJob.vue (hunk 3 above)
Comment (Mohit-Sony) [reply components/Careers/ApplyNewJob.vue]: have a look

File: components/Careers/ApplyNewJob.vue (hunk 4 above)
Comment (Mohit-Sony) [reply components/Careers/ApplyNewJob.vue]: done

File: components/Careers/ApplyNewJob.vue (hunk 6 above)
Comment (Mohit-Sony) [reply components/Careers/ApplyNewJob.vue]: email regs updated

File: components/Careers/ApplyNewJob.vue (hunk 5 above)
Comment (Mohit-Sony) [reply components/Careers/ApplyNewJob.vue]: i have use accept = ".pdf" this goes to form file picker 
then event of file upload would be handled by that file all changes are to be done there

**Pull request merged**